build/rollup-config.js

```diff
@@ -1,38 +1,53 @@
-import json from '@rollup/plugin-json';
 import {readFileSync} from 'node:fs';
-import rollupGitVersion from 'rollup-plugin-git-version';
```

Comment:
@IvanSanchez if we merge this, you can probably deprecate this package.

Reply:
This sounds like a good approach, I consider this resolved. @IvanSanchez wdyt?

Reply:
Marking this as resolved, the plugin was not used anyway, see https://github.com/Leaflet/Leaflet/pull/10011#issuecomment-3697212016